WordPress/src/main/res/values/strings.xml

```diff
@@ -1076,6 +1076,30 @@
 
     <!-- threat details -->
     <string name="threat_details">Threat Details</string>
+    <string name="threat_details_icon">Threat details icon</string>
+    <string name="threat_problem_header">What was the problem?</string>
+    <string name="threat_technical_details_header">The technical details</string>
+    <string name="threat_file_description">Threat found in file:</string>
+    <!-- threat fix -->
+    <string name="threat_fix_default_header">How will we fix it?</string>
+    <string name="threat_fix_fixed_header">How did Jetpack fix it?</string>
+    <string name="threat_fix_current_fixable_header">@string/threat_fix_default_header</string>
+    <string name="threat_fix_current_not_fixable_header">Resolving the threat</string>
+    <string name="threat_fix_current_not_fixable_description">Jetpack Scan cannot automatically fix this threat.
+        We suggest that you resolve the threat manually: ensure that WordPress, your theme, and all of your plugins are up to date, and remove the offending code, theme, or plugin from your site. \n \n
+        If you need more help to resolve this threat, we recommend <b>Codeable</b>, a trusted freelancer marketplace of highly vetted WordPress experts.
```

Comment:
🚫 We need to use html entities as all string tags are striped during the release process.

```suggestion
        If you need more help to resolve this threat, we recommend &lt;b&gt;Codeable&lt;/b&gt;, a trusted freelancer marketplace of highly vetted WordPress experts.
```